File: keiner_lindarte_aro/instructor_henry/mis_programas/ruby/orientado_a_objetos/5_clase_de_09_agosto/get_set.rb
class Bank_account
  attr_accessor :nombre, :identificacion, :dinero
  private :dinero

  def setDinero(dinero)
    self.dinero = dinero
  end

  def getDinero()
    return self.dinero
  end

  def initialize(nombre,identificacion)
    self.nombre = nombre
    self.identificacion = identificacion
  end

end

nuevo = Bank_account.new("keiner lindarte",1129539368)
nuevo.setDinero(10000)
puts nuevo.getDinero

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/python/clases_orientacion_objetos/5_clase_de_agosto_09/get_set.py
class Bank_account:
    nombre = ""
    identificacion = 0
    __dinero = 0
    
    def setDinero(self,dinero):
        self.dinero = dinero
        
    def getDinero(self):
        return self.dinero
    
    def __init__(self, nombre, identificacion):
        self.nombre = nombre
        self.identificacion = identificacion
        
nuevo = Bank_account("keiner lindarte",1129539368)
nuevo.setDinero(10000)
print(nuevo.getDinero())